src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2299,38 +2441,18 @@ export class StudyViewPageStore {
 
     public getCurrentBins(chartMeta: ChartMeta): number[] {
         if (this.isGeneSpecificChart(chartMeta.uniqueKey)) {
-            return _.uniq(
-                _.reduce(
-                    this.getGenomicChartDataBin(chartMeta).result,
-                    (acc, next) => {
-                        if (next.start) {
-                            acc.push(next.start);
-                        }
-                        if (next.end) {
-                            acc.push(next.end);
-                        }
-                        return acc;
-                    },
-                    [] as number[]
-                )
```

Comment:
@kalletlak I have another question about this part, we are using `if (next.start)` and `if (next.end)` at here. This will exclude the cases that `next.start === 0` and `next.end === 0`, is that what we want? Or we just need `start` and `end` are not `undefined`?

Reply:
I think this logic came from Hongxin originally, I just reused it. Looking at the function we need all non-zero numbers which is required for custom data bins, which looks correct to me.